salt/modules/cmdmod.py

```diff
@@ -52,6 +52,8 @@
 __proxyenabled__ = ['*']
 # Define the module's virtual name
 __virtualname__ = 'cmd'
+# Define blank context for importing for grains modules
+__context__ = {}
```

Comment:
This concerns me. I don't want an import of this module to overwrite the context if it's already been injected by the loader. Could we do this only if it hasn't already been set?